frontend/tests/test_perf_server_stage_frames.py

```
@@ -0,0 +1,588 @@
+
+    monkeypatch.setattr(cc, "llm_call_async", mutating_llm)
+
+    messages = _msgs(6)
+
+    async def _drive():
+        out = await cc.maybe_compact(sess, "url", "model", messages)
+        if cc._background_tasks:
+            await asyncio.gather(*list(cc._background_tasks))
+        return out
+
+    out_msgs, ctxlen, was = _run(_drive())
```

Ruff (0.15.20)

[warning] Unpacked variable `out_msgs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)